spec/resources/auth_spec.rb

```diff
@@ -10,69 +11,57 @@
     )
   end
 
-  it 'can use https' do
-    VCR.use_cassette('auth', match_requests_on: [:uri]) do
-      client = TableauApi.new(
-        host: ENV['TABLEAU_HTTPS_HOST'],
-        site_name: 'Default',
-        username: ENV['TABLEAU_ADMIN_USERNAME'],
-        password: ENV['TABLEAU_ADMIN_PASSWORD']
-      )
+  it 'can use https', vcr: vcr_options.merge(match_requests_on: [:uri]) do
+    client = TableauApi.new(
+      host: ENV['TABLEAU_HTTPS_HOST'],
+      site_name: 'Default',
+      username: ENV['TABLEAU_ADMIN_USERNAME'],
+      password: ENV['TABLEAU_ADMIN_PASSWORD']
+    )
 
-      client.connection.class.default_options.update(verify: false)
-      expect(client.auth.sign_in).to be true
-      client.connection.class.default_options.update(verify: true)
-    end
+    client.connection.class.default_options.update(verify: false)
+    expect(client.auth.sign_in).to be true
+    client.connection.class.default_options.update(verify: true)
   end
 
   # http://onlinehelp.tableau.com/v9.0/api/rest_api/en-us/help.htm#REST/rest_api_concepts_auth.htm%3FTocPath%3DConcepts%7C_____3
   # http://onlinehelp.tableau.com/v9.0/api/rest_api/en-us/help.htm#REST/rest_api_ref.htm#Sign_In%3FTocPath%3DAPI%2520Reference%7C_____51
-  it 'returns an instance of TableauApi::Resources::Auth' do
+  it 'returns an instance of TableauApi::Resources::Auth', vcr: vcr_options do
```

Comment:
Unfortunately in this file we need to define vcr per-spec/group, since we use different options for 'can use https'